### DISCUSSION TITLE
Problem with use PHPMailer

### DISCUSSION THREAD

**Discussion by iniacor** [vercel/community#2914] [Help]
### Summary

I got a 404 error when submitting my form. I use gulp to build the project, but because of this, the path to send_mail.php is lost. Tried all path options in this function async function sendData(data) {
return await fetch('../send_mail.php', {
method: 'POST',
body: data
});
} but nothing helps. this is link to repo https://github.com/iniacor/powercode and this is the project https://vercel.com/iniacor/powercode

### Steps to Reproduce

Fill form  with valid data and click submit button

**Comment by iniacor** [comment]
@amyegan  Hi! Could You helm me?

**Comment by amyegan** [comment]
Hi @iniacor! It looks like you recently updated your project structure to use an `api` directory. That would've been my first suggestion. 😄 

Did that solve the problem, or are you still having trouble?

**Comment by iniacor** [comment]
@amyegan  Hi) Thank You for feedback! Actually yes, i had to do it because in another way phpMailer did not work, it had not finded file send_mail.php. So i desided to go other way and set my output folder in gulp on root (./) But i still would like to know how set path to send_mail.php file that exist in api folder if i wiil use for my gulp build folder 'dist' or 'build' folder. What should i do in that case? Because i try all possible variants of path to send_mail.php file
